plugins/power/gpm-common.c

```diff
@@ -406,6 +406,12 @@ backlight_get_mock_value (enum BacklightHelperCommand command)
 	return ret;
 }
 
+static gboolean
+nvidia_driver_active (void)
+{
+        return g_file_test ("/sys/module/nvidia_drm", G_FILE_TEST_EXISTS | G_FILE_TEST_IS_DIR);
```

Comment:
If we have a screen context for these requests, is it possible to open a GLX context and then ask GLVND what driver is active for the screen? Just thinking ahead to possible multiple GPU cases, this tells us that there is an nvidia driver loaded, but not that it's the right backlight for the screen we were acting on.